posthog-android/src/main/java/com/posthog/android/internal/PostHogAndroidContext.kt

```diff
@@ -170,4 +172,20 @@ internal class PostHogAndroidContext(
 
         return dynamicContext
     }
+
```

Comment:
I was unsure whether to include the reference/inspiration, but the link is huge, and it looked odd in the code.

Reply:
Just add as a comment on top of the var